addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -137,7 +137,10 @@ export class AttachmentViewer extends Component {
      */
     _download() {
         const id = this.attachmentViewer.attachment.id;
-        this.env.services.navigate(`/web/content/ir.attachment/${id}/datas`, { download: true });
+        let downloadLink = document.createElement('a');
+        downloadLink.setAttribute('href', `/web/content/ir.attachment/${id}/datas`);
+        downloadLink.setAttribute('download','');
```

Comment:
This is the way how HTML5 works with the download, `<a href=url download>`. Without the `download` attribute, it will not work. The value of the `download` attribute specifies the rename of the file, in our case, it is set to empty so that the original filename will be used.
ref: https://developers.google.com/web/updates/2011/08/Downloading-resources-in-HTML5-a-download

Reply:
Thank you, I didn't know about this param.

But I'm not conformtable with introduce a new way of doing it during a fix?

Normally that's what our `download=True` parameter does in the URL (see `/web/content` route), without requiring browser support of the attribute?

```
if download:
            headers.append(('Content-Disposition', content_disposition(filename)))
```